### PR TITLE
Testing #433 with solved merge conflicts

### PR DESCRIPTION
**Pull request by kacf**

### CODE REVIEW

**Comment by kacf** [comment]
@lluiscampos: I have already reviewed the code from #433, but maybe you want to take a quick look at my supporting commits?

**Comment by kacf** [comment]
@sensslen: Perhaps you want to take a look at the last commit in particular, and make sure your use case is still covered.

File: .gitlab-ci.yml [hunk 1]
```diff
@@ -275,6 +288,28 @@ test_acceptance_ubuntu:
...
+  tags:
+    - docker
+  image: docker
+  services:
+    - name: docker:20.10.8-dind
+      alias: docker
+      command: ["--mtu=1440"] # https://gitlab.com/gitlab-com/gl-infra/production/-/issues/5590
+  before_script:
+    - docker login -u $DOCKER_HUB_USERNAME -p $DOCKER_HUB_PASSWORD
+    - docker import image.tar $DOCKER_REPOSITORY:pr
```
Comment (lluiscampos) [inline .gitlab-ci.yml:302]: Why not `docker load`? With load we will preserve the image tag
```suggestion
    - docker load -i image.tar
```

**Review by lluiscampos — APPROVED** [review]

File: .gitlab-ci.yml (hunk 1 above)
Comment (kacf) [reply .gitlab-ci.yml:302]: Yes, much better! I will change that.

**Comment by sensslen** [comment]
> @sensslen: Perhaps you want to take a look at the last commit in particular, and make sure your use case is still covered.

Looks good to me.

**Pull request merged**